### PR TITLE
Removed font-awesome from the admin section

### PR DESCRIPTION
**Pull request by nickytonline**
## What type of PR is this? (check all applicable)

- [x] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

We're looking to remove or at a minimum load Bootstrap conditionally. While looking into this, I noticed that font-awesome was used. I narrowed it down to two spots that I replaced with our standard icons.

## QA Instructions, Screenshots, Recordings

1. Login as admin@forem.local, password: password

### Comments Admin page

1. Load the comments page for the admin, `admin/content_manager/comments`.
2. Comments should look pretty identical. The new Twitter icon is a little bigger but it's our standard one. 

**Before**

image

**After**

image

Note that the font is smaller now because an `<h5 />` is not being used. The spacing is a little less too for the same reason. I think the size is fine, but let me know @ludwiczakpawel.

### Flash messages in the admin

1. Update something to trigger the flash message, e.g. edit a user, `http://localhost:3000/admin/users/1/edit`
2. The success or failure message appears with an X button.
3. The X icon should pretty much be the same as what it was before as it's using out standard X icon now.

**Before**

image

**After**

image

### UI accessibility concerns?

I replaced an `<h5 />` tag with a `<div />` tag around the links for a comment as it made no sense.

## Added/updated tests?

- [ ] Yes
- [x] No, and this is why: This is only an aesthetic change.
- [ ] I need help with writing tests

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

- [ ] I've updated the [Developer Docs](https://developers.forem.com) or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
- [ ] This PR changes the Forem platform and our documentation needs to be
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
- [ ] I've updated the README or added inline documentation
- [ ] I've added an entry to
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
- [ ] I will share this change in a [Changelog](https://forem.dev/t/changelog)
      or in a [forem.dev](http://forem.dev) post
- [ ] I will share this change internally with the appropriate teams
- [ ] I'm not sure how best to communicate this change and need help
- [x] This change does not need to be communicated, and this is why not: _please
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

File: app/views/admin/comments/index.html.erb [hunk 1]
```diff
@@ -16,19 +16,19 @@
                          attributes: %w[href strong em ref rel src title alt class] %>
 
           </p>
-          <h5 class="card-title text-center">
```
Comment (nickytonline) [inline app/views/admin/comments/index.html.erb:19]: An `<h5 />` made no sense here.

**Review by citizen428 — APPROVED** [review]

**Comment by ludwiczakpawel** [comment]
> Note that the font is smaller now because an <h5 /> is not being used. The spacing is a little less too for the same reason. I think the size is fine, but let me know @ludwiczakpawel.

It's like when you borrow a doll from your younger sister, and you completely ruin it, like put hair on fire, break some limbs, make a hole in the head etc - this is Admin panel.
So you gave it back to your sister, she's obviously mad at you but she's like _"I'm gonna use mom's lipstick to fix her smile"_ and yea the smile is probably a bit better (your sis is 3y old and she can barely draw or paint anything) but everything else is still terrible and it's still ugly and broken doll. But it has a nice smile! 

And that lipstick is removing font awesome. Our doll still looks bad but removing font awesome makes it a tiny bit better :)  

So I'm definitely approving this change! :)

**Review by ludwiczakpawel — COMMENTED** [review]
> Note that the font is smaller now because an <h5 /> is not being used. The spacing is a little less too for the same reason. I think the size is fine, but let me know @ludwiczakpawel.

It's like when you borrow a doll from your younger sister, and you completely ruin it, like put hair on fire, break some limbs, make a hole in the head etc - this is Admin panel.
So you gave it back to your sister, she's obviously mad at you but she's like _"I'm gonna use mom's lipstick to fix her smile"_ and yea the smile is probably a bit better (your sis is 3y old and she can barely draw or paint anything) but everything else is still terrible and it's still ugly and broken doll. But it has a nice smile! 

And that lipstick is removing font awesome. Our doll still looks bad but removing font awesome makes it a tiny bit better :)  

---

Also, are you sure we don't use it anywhere else? https://cs.github.com/?scope=org%3Aforem&scopeName=forem&q=%22fa+fa-%22

Lets confirm that and then I can see no blockers for approving :)

**Review by aitchiss — APPROVED** [review]
Looks good to me! I searched for any other uses of font awesome but any `fa-` references I found were regexes, and had nothing to do with font awesome 🌈

**Comment by nickytonline** [comment]
> Lets confirm that and then I can see no blockers for approving :)

@ludwiczakpawel these were the only two places I found font awesome being used. All other instances of fa- (don't awesome prefix) were in regexes or code not related to font awesome like @aitchiss mentioned.

**Review by ludwiczakpawel — APPROVED** [review]
Thanks!

**Review by juliannatetreault — APPROVED** [review]

**Pull request merged**